ucsmsdk/utils/comparesyncmo.py

```diff
@@ -352,14 +352,14 @@ def compare_ucs_mo(ref_obj, diff_obj,
                                   compare mos only which exist on both
```

Comment:
default:False. When True, compares MOs that exist on both reference and difference UCSM